### DISCUSSION TITLE
Two page layout

### DISCUSSION THREAD

**Discussion by bilalmalkoc** [t1m0n/air-datepicker#605] [Q&A]
Hello. Is it possible two page calendar layout? It is more useful for daterange selection.

<img width="665" alt="68747470733a2f2f6465762e6a7378342e636f6d2f656173657069636b2f73637265656e73686f742e706e67" src="https://github.com/t1m0n/air-datepicker/assets/23140025/3d7d395a-e764-4f8b-9732-cbdd33081da4">

**Comment by t1m0n** [comment]
Hi! Unfortunately no. This feature is planned, but I can't say when it will be released

**Reply by YanTomchik** [reply]
It is very useful feature, i agree). Maybe there is information about when it will be added?

**Comment by tanya-avtey** [comment]
Hi! Should we expect this add-on in the near future?
